### PR TITLE
feat: add conference albums

### PR DESCRIPTION
**Pull request by jack482653**
This PR has features below:

1. Disable Call For Proposal page link at navigation bar.
2. Remove / disable register button.
3. Add Album page.

### CODE REVIEW

**Comment by yungyuc** [comment]
It will be great if the PR includes a summary in the opening comment box.

File: content/2023/configurations/banners.tsx [hunk 1]
```diff
@@ -14,11 +14,11 @@ const showBannerPageConfigs: ShowBannerPageConfigType[] = [
       title: 'sciwork conference',
       subTitle:
         'December 9th to 10th\nNational Yang Ming Chiao Tung University',
-      children: (
-        <Button to={REGISTER_URL} target="_blank" variant="action">
-          Register
-        </Button>
-      ),
+      // children: (
```
Comment (yungyuc) [inline content/2023/configurations/banners.tsx:17]: I do not know about frontend coding.  But as a general rule of thumb it is not a good style to use comments for disabling dead code in production.

**Review by yungyuc — APPROVED** [review]

**Comment by jack482653** [comment]
> It will be great if the PR includes a summary in the opening comment box.

I have filled the description of this PR. Thanks for remind.

File: content/2023/configurations/banners.tsx (hunk 1 above)
Comment (jack482653) [reply content/2023/configurations/banners.tsx]: It is not necessary to keep code. Removed at 4dd77e72507c8a8e17446a50b68bc6cde5727a25

**Pull request merged**